packages/components/app/styles/components/dropdown.scss

```diff
@@ -408,8 +416,82 @@ $hds-dropdown-toggle-border-radius: 5px;
   }
 }
 
+// HDS::DROPDOWN::LIST-ITEM::TITLE
+
 .hds-dropdown-list-item--title {
   padding: 10px 16px 4px;
   color: var(--token-color-foreground-strong);
 }
 
+// HDS::DROPDOWN::LIST-ITEM::CHECKMARK
+
+.hds-dropdown-list-item--checkmark {
+  color: var(--token-color-foreground-action-active);
+
+  a.hds-dropdown-list-item__interactive,
+  button.hds-dropdown-list-item__interactive {
+    // leave room for the checkmark to avoid overlapping with text
+    padding-right: 30px;
+
+    // disable background on focus and active states
+    &:focus,
+    &:active,
+    &.mock-focus,
+    &.mock-active {
+      &::after {
+        background-color: inherit;
+      }
+    }
+  }
+
+  .hds-dropdown-list-item__interactive-text {
+    flex: 1;
+  }
+}
+
+.hds-dropdown-list-item--checkmark-selected {
+  background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M14.78 4.28a.75.75 0 00-1.06-1.06l-7.97 7.97-3.47-3.47a.75.75 0 00-1.06 1.06l4 4a.75.75 0 001.06 0l8.5-8.5z' fill='%231060ff'/%3E%3C/svg%3E");
+  background-repeat: no-repeat;
+  background-position: top 10px right 10px;
+  background-size: var(--token-form-select-background-image-size) var(--token-form-select-background-image-size);
+
+  .hds-dropdown-list-item__interactive {
```

Comment:
maybe it's me, but may be that this rule is not applied? I don't see it in the styles when I inspect it in the browser (but may be doing something wrong).

Reply:
yes, it wasn't applied, because of lack of specificity – adjusted now